utils/firestore.js

```
@@ -530,7 +578,8 @@ const retrieveParticipantsDemo = async (siteCode, type, isParent, limit, cursor,
             }
 
             if (cursor) {
-                const doc = await getCursorDocumnet('participants', cursor);
+                const collection = 'participants';
+                const doc = await getCursorDocument(collection, cursor);
```

I think doc may either exist or be an `instanceof Error` here if the cursor isn't found.

---

can you clarify what you mean here?

---

My mistake...the else branch handles it. I'm wondering if we should return that new Error from the else block so the query doesn't execute?